### PR TITLE
Fix PWM not working in Raspi 4

### PR DESCRIPTION
**Pull request by rami-dabain**
Minor permission problems

### CODE REVIEW

**Comment by baumgarb** [comment]
Nice work man, just tried it out and it fixes the issue. Sweet! :)

**Comment by rami-dabain** [comment]
Looks like this library is not maintained anymore @stianeikeland  i can help maintain this if you give me write access to this repo :)

File: rpio.go [hunk 1]
```diff
@@ -222,6 +222,12 @@ func (pin Pin) DutyCycle(dutyLen, cycleLen uint32) {
 	SetDutyCycle(pin, dutyLen, cycleLen)
 }
 
+// DutyCycle: Set duty cycle for Pwm pin (see doc of SetDutyCycle)
+func (pin Pin) DutyCyclePercentage(freq int, dutyPercentage uint32) {
+	SetFreq(pin, freq * 100)
+	SetDutyCycle(pin, dutyPercentage, 100)
+}
+
```
Comment (sagikazarmark) [inline rpio.go:230]: If this PR is about fixing a specific issue, it shouldn't contain new features.

**Review by sagikazarmark — CHANGES_REQUESTED** [review]

**Comment by Peelz** [comment]
Is this PR will merged ?

**Comment by drahoslove** [comment]
@rami-dabain Fixed it myself in another commit because your master branch contained also an additional feature not relate to the issue, as mentioned in the review comment.

**Pull request closed without merging**